src/scripts/createStore.js

```diff
@@ -0,0 +1,23 @@
+import { createStore, applyMiddleware, compose } from "redux";
+import thunk from "redux-thunk";
+
+import reducer from "reducers/index";
+
+const composeEnhancers = /* window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || */ compose;
```

Comment:
When we want to use the redux-devtools chrome extension, simply uncomment `window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__`